src/devices/cpu/i386/i386ex.cpp

```diff
@@ -0,0 +1,316 @@
+#include "i386ex.h"
```

Comment:
This file is missing the standard header comment, and the PCH.

Reply:
what is PCH?